spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/TaskCommands.java

```diff
@@ -213,6 +219,12 @@ public String launch(
 		return String.format("Launched task '%s' with execution id %d", name, taskExecutionId);
 	}
 
+	@CliCommand(value = STOP, help = "Stop executing tasks")
+	public String stop(@CliOption(key = { "", "ids" }, help = "the task execution id", mandatory = true) String ids) {
+		taskOperations().stop(ids);
+		return String.format("Request to stop task execution %s has been submitted", ids);
```

Comment:
Since there are multiple ids, it might be better to use:

Request to stop task execution(s) [%s] has been submitted

I can fix this on merge if you are ok with this.

Reply:
It seems the way how it is logged would be correct at runtime, though. For instance, if the IDs passed were, let's say: `22, 45, 33`, then the return would be:

 > Request to stop task execution 22, 45, 33 has been submitted"

Perhaps we could change it to:

 > Request to stop the task execution with id(s): 22, 45, 33 has been submitted"

Reply:
yeah, this is better!

Reply:
👍